tests/transformers/test_modeling.py

```diff
@@ -12,13 +12,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import unittest
-from paddlenlp.transformers import TinyBertModel
+import shutil
+from tempfile import TemporaryDirectory
+from tests.testing_utils import slow
+from multiprocessing import Pool
+from paddlenlp.transformers import TinyBertModel, BertModel
+from paddlenlp.utils.env import MODEL_HOME
+
+
+def download_bert_model(model_name: str):
+    """set the global method: multiprocessing can not pickle local method
+
+    Args:
+        model_name (str): the model name
+    """
+
+    model = BertModel.from_pretrained(model_name)
+    # free the model resource
+    del model
 
 
 class TestModeling(unittest.TestCase):
     """Test PretrainedModel single time, not in Transformer models"""
 
+    @slow
     def test_from_pretrained_with_load_as_state_np_params(self):
         """init model with `load_state_as_np` params"""
         TinyBertModel.from_pretrained("tinybert-4l-312d", load_state_as_np=True)
```

Comment:
can we add a simple model forward and a simple assert here just to make sure the model load is indeed valid?